File: backend/routers/risk.py
"""Module 6 risk API routes."""
from fastapi import APIRouter
from schemas.risk import RiskResponse
from services.risk_engine import build_risk_response

router = APIRouter(prefix="/applications", tags=["Risk"])


@router.post("/{app_id}/assess-risk", response_model=RiskResponse)
async def assess_risk(app_id: str, verification_result: dict):
    """Receive Module 5 verification JSON and return risk assessment."""
    return build_risk_response(app_id, verification_result)

File: frontend/src/api/risk.js
/**
 * Risk API
 */

import api from './client';

const BASE = '/api/v1';

export const getFlags = (appId) =>
  api.get(`${BASE}/applications/${appId}/flags`);

export const assessRisk = (appId) =>
  api.post(`${BASE}/applications/${appId}/assess-risk`);
